### PR TITLE
fix: Update quickstart gallery to be consistent with documentation

### PR DESCRIPTION
**Pull request by ssheng**
Update the quickstart gallery project to the consistent with the quickstart documentation listed here, https://docs.bentoml.org/en/latest/quickstart.html.

### CODE REVIEW

File: quickstart/iris_classifier.ipynb [hunk 1]
```diff
@@ -187,14 +187,7 @@
...
    "source": [
-    "bentoml.build(\n",
-    "    \"iris_classifier.py:svc\",\n",
-    "    include=[\"*.py\"],\n",
-    "    description=\"file:./README.md\",\n",
-    "    python=dict(\n",
-    "        packages=[\"scikit-learn\"]\n",
-    "    )\n",
-    ")"
+    "!bentoml build"
```
Comment (parano) [inline quickstart/iris_classifier.ipynb:190]: this unfortunately doesn't work in google colab

Comment (ssheng) [reply quickstart/iris_classifier.ipynb:190]: What's the problem? Is it worthwhile fixing from our side?

**Comment by aarnphm** [comment]
@ssheng can you also change the port from 5000 to 3000?

**Comment by parano** [comment]
> @ssheng can you also change the port from 5000 to 3000?

there's already a pending PR for that

**Pull request merged**